website/docs/r/prometheus_workspace.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
+subcategory: "Amazon Managed Service for Prometheus (AMP)"
+layout: "aws"
+page_title: "AWS: aws_prometheus_workspace"
+description: |-
+  Creates a prometheus workspace
+---
+
+# Resource: aws_prometheus_workspace
+
+Provides a managed prometheus workspace.
+
+-> **Note:** only workspace alias can be updated today
```

Comment:
We have found this types of documentation notes hard to maintain over time (easy to forget to change). Instead this resource documentation verbiage should include a note about the AWS service being in preview.

```suggestion
~> **NOTE:** This AWS functionality is in Preview and may change before General Availability release. Backwards compatibility is not guaranteed between Terraform AWS Provider releases.
```

Reply:
👍